Public/JS/pg.js

```diff
@@ -0,0 +1,9 @@
+ let formname=document.querySelector("#form_name");
+ let email = document.querySelector("#form_email");
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/sanchezabad034/CALIDADSOFTWARE/pullRequest?prid=9675051)
```suggestion
let email = document.querySelector("#form_email");
```